lib/model/surveys_model.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:equatable/equatable.dart';
+import 'package:flutter_survey/api/response/surveys_response.dart';
+import 'package:flutter_survey/model/survey_model.dart';
+
+class SurveysModel extends Equatable {
+  final List<SurveyModel> surveys;
+
+  const SurveysModel({
+    required this.surveys,
+  });
```

Comment:
Do we need to have this model? Why don't we use `List<SurveyModel>` as a type in other places to simplify this?

Reply:
You're right, it was a bit too redundant, I have removed it 🙏